vault/dendron.topic.special-notes.commands.create-daily-journal-note.md

```diff
@@ -0,0 +1,19 @@
+---
+id: 9stxbnf9hxmxt76se5dtx2b
+title: "Dendron: Create Daily Journal Note"
+desc: "Create a daily journal note."
```

Comment:
as a convention, can we not include the period in the frontmatter? if we want to add a period, we can add it afterwards